task/03-date-tasks.js

```diff
@@ -56,10 +56,21 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+   const year = date.getFullYear();
+   let isLeap;
+   if(year % 4 !== 0){
+      isLeap = false;
```

Comment:
You can return `true/false` directly.

Reply:
Also your `year % 100 !== 0` expression returns a logic result. Your function can return a result without `if`.